dev-embedded/ponyprog/files/ponyprog-3.0.0-install_doc.patch

```diff
@@ -0,0 +1,36 @@
+--- ponyprog-3.0.0/distribution/posix/CMakeLists.txt-	2018-01-18 21:12:21.504884644 +0300
++++ ponyprog-3.0.0/distribution/posix/CMakeLists.txt	2018-01-18 22:02:34.575994692 +0300
+@@ -33,25 +33,25 @@
+         ) 
+ 
+ INSTALL(DIRECTORY "${PROJECT_SOURCE_DIR}/distribution/innosetup/" 
+-        DESTINATION "share/doc/ponyprog/" 
++        DESTINATION "share/doc/ponyprog-3.0.0/" 
+         COMPONENT "help files" 
+         DIRECTORY_PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ GROUP_EXECUTE GROUP_READ WORLD_EXECUTE WORLD_READ 
+         FILES_MATCHING PATTERN "*.jpg" PATTERN "*.html"
+         ) 
+ 
+ INSTALL(FILES "${PROJECT_SOURCE_DIR}/copyright" 
+-        DESTINATION "share/doc/ponyprog/" 
++        DESTINATION "share/doc/ponyprog-3.0.0/" 
+         COMPONENT "application" 
+         PERMISSIONS OWNER_WRITE OWNER_READ GROUP_READ WORLD_READ
+         ) 
+ # MESSAGE("PROJECT_SOURCE_DIR ${PROJECT_SOURCE_DIR}")
+ 
+-INSTALL(FILES "${PROJECT_SOURCE_DIR}/desktop/ponyprog.desktop"
+-        DESTINATION "share/applications/" 
+-        COMPONENT "application" 
+-        PERMISSIONS OWNER_WRITE OWNER_READ GROUP_READ WORLD_READ
+-        )
+-        
++#INSTALL(FILES "${PROJECT_SOURCE_DIR}/desktop/ponyprog.desktop"
++#        DESTINATION "share/applications/" 
++#        COMPONENT "application" 
++#        PERMISSIONS OWNER_WRITE OWNER_READ GROUP_READ WORLD_READ
```

Comment:
and remove this permission crap, its stupid and causes issues for us

Reply:
remaked. but  ${CMAKE_INSTALL_DOCDIR} has no version... must be .../usr/share/doc/ponyprog-3.0.0

Reply:
Yeah, you need to pass `-DCMAKE_INSTALL_DOCDIR` to `mycmakeargs` manually.

Reply:
added